server.js

```diff
@@ -64,6 +64,7 @@ server.use((req, res, next) => {
           console.log(
             `Error: ${err},
             Request: ${req.url},
+            Host: ${req.get('host')},
```

Comment:
Label 'Host' on $ statement.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Expected an identifier and instead saw ','.